### PR TITLE
ENDOC-300 update external DB docs

### PR DESCRIPTION
**Pull request by joewhite101**

### CODE REVIEW

File: vuepress/docs/next/tutorials/devops/external-database/README.md [hunk 1]
```diff
@@ -20,40 +20,100 @@ backup workflow that will scale with your application.
...
-        the Entando app and any services that use it
-
-3.  Deploy your app
-
-
+1. Create and install a secret or secrets for your database credentials
+2. Edit the deployment template generated from running the helm command or the helm template you used to deploy your Entando application
+3. Find the entry for the `EntandoCompositeApp`
+4. Set the value for `dbms` to `none`
+5. Under `environmentVariables` you will need to provide database connection information for two Entando Server database and one connection for the Entando Component Repository. Those connects  are:
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/external-database/README.md:27]: 1) databases, plural. 2) "Those connects" is a little awkward.

File: vuepress/docs/next/tutorials/devops/external-database/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This document provides a guide to connecting to an external database. In
+This document provides a guide to connecting an Entando Application to an external database. In
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/external-database/README.md:5]: Should these edits be applied to v6.3.0 as well?

**Review by nshaw — CHANGES_REQUESTED** [review]

File: vuepress/docs/next/tutorials/devops/external-database/README.md (hunk 2 above)
Comment (joewhite101) [reply vuepress/docs/next/tutorials/devops/external-database/README.md:5]: Yeah. I think the quickstart version is still accurate there but this is more comprehensive for most use cases.

File: vuepress/docs/next/tutorials/devops/external-database/README.md (hunk 1 above)
Comment (joewhite101) [reply vuepress/docs/next/tutorials/devops/external-database/README.md]: Done.

File: vuepress/docs/next/tutorials/devops/external-database/README.md (hunk 2 above)
Comment (joewhite101) [reply vuepress/docs/next/tutorials/devops/external-database/README.md:5]: Done.

**Review by nshaw — APPROVED** [review]

**Pull request merged**